librepo/yum.c

```diff
@@ -152,9 +162,29 @@ lr_yum_repomd_record_enabled(LrHandle *handle, const char *type)
                 return TRUE;
             x++;
         }
+        // Substitution check
+        if (handle->yumslist) {
+            LrUrlVars* elem = handle->yumslist;
+            while (TRUE) {
```

Comment:
Instead of this two lines (167-168) + lines 181-184, I would prefer:

    for (GSList *elem = repomd->records; elem; elem = g_slist_next(elem)) {

which does exactly the same thing, but it's more compact and readable